.github/dependabot.yml

```diff
@@ -0,0 +1,37 @@
+# Automatically open PRs to update outdated deps
+# See https://docs.github.com/en/github/administering-a-repository/enabling-and-disabling-version-updates
+
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "cargo"
+    # Look for Cargo `.toml` and `.lock` files in the `root` directory
+    directory: "/"
+    # Check the cargo registry for updates every day (weekdays)
+    schedule:
+      interval: "daily"
```

Comment:
Maybe we can do it weekly? The repo is still going through breaking changes all the time, and we have a chronic issue with too many outstanding PRs, so would be nice to minimize the amount of work we get from the bot.

Reply:
Sure, however you'd prefer! One thing to note: configuring it to only check weekly won't reduce the amount of work required, it'll just mean all the work shows up at once on the same day ;) 

Regarding the workload, it should be minimal in any case. Usually you just either merge in the PRs if CI passes, or postpone the update (via a special comment on the PR or just closing the PR) and create an issue to fix the compatibility with the library.

However, if updated or dealing with outdated dependencies isn't something that's an issue, then maybe you don't need this enabled here! :)